### DISCUSSION TITLE
Image hero with text

### DISCUSSION THREAD

**Discussion by Fedora-Man** [holger1411/understrap#1691] [Q&A]
What is the best way to create an image hero with text using bootstrap in understrap?

**Comment by bacoords** [comment]
I recommend examining the source code of their examples on the Bootstrap site, especially the 'jumbotron' example: https://getbootstrap.com/docs/5.1/examples/#custom-components

**Reply by Fedora-Man** [reply]
jumbotron is deprecated in BS5 and the examples don't have an example for and image as background and text on top.

**Reply by bacoords** [reply]
Correct the Jumbotron has been deprecated, but the demo is meant to show how you can replicate it with other classes.

As for the background image, you can use a css background-image or try an image with the position-absolute utility class
